assets/src/components/notificationModals/basicNotificationModal.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react"
+import { Modal } from "@restart/ui"
+import { setNotification } from "../../state"
+import { StateDispatchContext } from "../../contexts/stateDispatchContext"
+import { OldCloseIcon } from "../../helpers/icon"
+
+const BasicNotificationModal = ({
```

Comment:
Commenting a bit late here, but did you mean for this to be `Basic` or `Base`?

I only ask because the title of the PR says `Base`, and that's what I would normally have expected.